plugins/media/wowza/services/LiveConversionProfileService.php

```diff
@@ -207,6 +208,17 @@ private function calculateFlavorHeight($flavorResolution, $ingestParameters)
 		return ($ingestParameters[self::WIDTH] != 0) ? (($flavorResolution[self::WIDTH] * $ingestParameters[self::HEIGHT]) / $ingestParameters[self::WIDTH]) : 0;
 	}
 	
+	private function checkFlavorsDataRate($ingestDataRate, $flavorDataRate)
+	{
+		$percentageFactor = 1 + (self::VIDEO_DATA_RATE_PERCENTAGE_GAP_BETWEEN_FLAVORS / 100);
+		return ($ingestDataRate !== 0) && (($ingestDataRate * 1024) < ($flavorDataRate * $percentageFactor));
```

Comment:
change 1024=>1000
http://io9.gizmodo.com/is-a-kilobit-1-000-or-1-024-bits-a-mathematical-debat-1694610423